src/html/partials/behind-the-scenes.html

```diff
@@ -1,4 +1,57 @@
 <section class="behind-scenes-section">
+
   <h2 class="sm-title">Behind The Scenes</h2>
-  <p class="bg-intro">While Florence is only a concept, here are some things that will be helpful in making her come alive.</p>
-</section>
+  <p class="lg-text">While Florence is only a concept, here are some things that will be helpful in making her come alive.</p>
+
+  <!-- <div class="behind-scenes-section__florence">
+    <div class="behind-scenes-section__florence__book"></div>
+    <div class="behind-scenes-section__florence__gear"></div>
+    <div class="behind-scenes-section__florence__comment"></div>
+    <div class="behind-scenes-section__florence__heart"></div>
+    <div class="behind-scenes-section__florence__link"></div>
+  </div> -->
+
+  <ul class="behind-scenes-section__list">
+    <li class="behind-scenes-section__list__item">
+      <button class="behind-scenes-section__list__item__btn--book"></button>
```

Comment:
For accessibility purposes, the `button` is the right approach, but we'll need some sort of text to go with it. The simplest option is to add the `aria-label` attribute, something like: `aria-label="Show Info"`. Screen readers will read this as "Show Info button". You could also add more context in the label: `aria-label="Show Medical Knowledge Base Info"` (read as "Show Medical Knowledge Base Info button").